napari/utils/colormaps/colormap.py

```diff
@@ -206,7 +206,15 @@ def map(self, values):
                 if len(color) == 3:
                     color = np.append(color, 1)
                 mapped[idx] = color
+            else:
+                mapped[idx] = self.default_color
         # If using selected, disable all others
         if self.use_selection:
             mapped[~np.isclose(values, self.selection)] = 0
         return mapped
+
+    @property
+    def default_color(self):
+        if self.use_selection:
+            return (0, 0, 0, 0)
+        return self.color_dict.get(None, (0, 0, 0, 0))
```

Comment:
In my opinion it would be nicer to have some sort of grey color (maybe semitransparent) for unset labels, at least by default.

Reply:
real color is set in `Labels.color` setter.